VideoApi/VideoApi.IntegrationTests/Database/Queries/GetConferenceInterpreterRoomsByDateQueryTests.cs

```diff
@@ -0,0 +1,95 @@
+using FluentAssertions;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using VideoApi.Domain.Enums;
+using VideoApi.DAL;
+using VideoApi.DAL.Queries;
+using VideoApi.Domain;
+using System.Linq;
+
+namespace VideoApi.IntegrationTests.Database.Queries
+{
+    public class GetConferenceInterpreterRoomsByDateQueryTests :DatabaseTestsBase
+    {
+        private GetConferenceInterpreterRoomsByDateQueryHandler _handler;
+        private Guid _newConferenceId;
+        private List<Room> _rooms;
+        private List<Event> _events;
+
+        [SetUp]
+        public void Setup()
+        {
+            _newConferenceId = Guid.Empty;
+            var context = new VideoApiDbContext(VideoBookingsDbContextOptions);
+            _handler = new GetConferenceInterpreterRoomsByDateQueryHandler(context);
+        }
+
+        [Test]
+        public async System.Threading.Tasks.Task Should_return_an_empty_list_if_conference_does_not_exist()
```

Comment:
Add using instead

Reply:
there is a conflict hence had a fully qualify. DAL in video api as task domain object.